### ISSUE TITLE
Update on the Docker hub

### ISSUE THREAD

**Issue by rdavid** [ttyS0/docker-other-transcode#3]
Please, update the image at the Docker hub:
 https://registry.hub.docker.com/r/ttys0/other-transcode/
 
It is the only way to get the latest on Synology's Docker application.

Thank you! I'm a fan of the Don Melton's transcode tool for many years.

**Comment by ttyS0** [comment]
Thanks for using these images! 

I'd be very surprised if Synology can't use anything other than Docker's registry. I've actually been planning on pulling all my containers off of Docker Hub due to their recent policy changes. 

I don't have a Synology, but would expect it to be able to use another public registry, such as GitHub, `ghcr.io/ttys0/other-transcode:version`, which is where the images for these containers have been for a long time now. 

Do you get an error when you try to pull an image from GitHub?

`docker pull ghcr.io/ttys0/other-transcode:qsv-latest`

**Comment by rdavid** [comment]
Personally, I prefer `ghcr.io` as well. Synology supports its own registry, see the attached image. 
<img width="981" alt="Screen Shot 2023-04-08 at 23 03 52" src="https://user-images.githubusercontent.com/164628/230740528-83e367cd-f343-4ce3-a65a-58ace8af06a3.png">

**Comment by rdavid** [comment]
Sure, I can run my own container on the Synology, that was my thinking was an hour ago. But you have created and maintained the extension of the beautiful tool. I'd like to use it on the Synology.

**Comment by ttyS0** [comment]
Ahhh, I assumed the container stuff was handled via a CLI, so you could just pull the prebuilt image from GitHub. Honestly, I would expect to be able to do that via a web UI as well. I've never used a Synology, however, so I suppose they may assume Docker Hub is the only registry around. 

I went ahead and pushed the `-latest` images to Docker Hub. I'll update the GitHub Action that builds them to also push there for future builds.

**Comment by ttyS0** [comment]
@rdavid Were you able to pull the updated images from Docker Hub?

**Comment by rdavid** [comment]
@ttyS0 Yes, I've pulled the updated image on the Synology, thank you. I'm investigating, what kind of hardware transcoding the Synology DS1019+ is capable of. Together with this, I learn about `other-transcode` in general. I've been using `[video_transcoding](https://github.com/donmelton/video_transcoding)` for a decade, I wrote [my own helper](https://github.com/rdavid/transcode), but I'm new to the new tool.
